### PR TITLE
Fix rate calculation

### PR DESCRIPTION
**Pull request by desofto**

### CODE REVIEW

**Comment by boblin** [comment]
@desofto: spec are failing. Can you fix that?

File: spec/cnb/monthly_rates_spec.rb [hunk 1]
```diff
@@ -27,11 +27,11 @@
 
     context 'valid parameters' do
       it 'returns rate for given currency and date' do
-        expect(monthly_rates.rate_for(currency, Date.today)).to eql(19.564)
+        expect(monthly_rates.rate_for(currency, Date.today)).to eql(0.19564)
```
Comment (desofto) [inline spec/cnb/monthly_rates_spec.rb:30]: Fixed. Fix Date.today too?

**Comment by boblin** [comment]
@desofto please, rebase with master.

**Comment by desofto** [comment]
Rebased

**Comment by boblin** [comment]
@desofto what is motivation for this merge request? 

Why did you changed specs results? @Masa331 are they OK?

**Comment by desofto** [comment]
When it calculated exchange rate it did not care on Sum for exchange in local currency. Tests in Fantozzi failed, I found out why and fixed it :)

File: spec/cnb/monthly_rates_spec.rb [hunk 2]
```diff
@@ -27,11 +27,11 @@
...
       end
 
       it 'returns rate for given currency with default date' do
-        expect(monthly_rates.rate_for(currency)).to eql(19.564)
+        expect(monthly_rates.rate_for(currency)).to eql(0.19564)
```
Comment (Masa331) [inline spec/cnb/monthly_rates_spec.rb:34]: so this is tested on JPY? Because thats only currency i know which has 100 quantity..

Comment (desofto) [inline spec/cnb/monthly_rates_spec.rb:34]: If this question to me, in Fantozzi we expected this:

``` ruby
expect(ExchangeRate.find_by(currency_pair_id: jpy_czk.id, date: '24.10.2015').rate).to eq 0.20244
```

**Comment by Masa331** [comment]
Yep, it's ok @boblin please merge it

**Pull request merged**